esp32/modules/appglue.py

```diff
@@ -2,11 +2,12 @@
 
 def start_app(app):
     ugfx.clear(ugfx.WHITE)
-    ugfx.string(0, 0, "Please wait...", "PermanentMarker22", ugfx.BLACK)
+    ugfx.string(0, 0, "Rebooting..", "PermanentMarker22", ugfx.BLACK)
     if (app==""):
         ugfx.string(0,  25, "Returning to homescreen...","Roboto_Regular12",ugfx.BLACK)
     else:
         ugfx.string(0,  25, "Starting "+app+"...","Roboto_Regular12",ugfx.BLACK)
+    ugfx.set_lut(ugfx.LUT_FASTEST)
```

Comment:
You don't have to `set_lut` separately, you can pass `ugfx.LUT_FASTEST` to `ugfx.flush()`